static.config.js

```diff
@@ -160,12 +169,13 @@ export default {
         component: 'src/js/pages/Home',
         children: allPages('ar'),
         getData: async () => ({
+          langCode: "ar",
           topServices: topServices_ar,
         }),
       },
       {
         is404: true,
-        component: 'src/js/pages/404',
+        component: 'src/js/pages/404', //TODO: update 404 page to be conscious of all languages
```

Comment:
Good catch. We should get this on the content team's radar as an issue. We'll need the language a user should see when they 404. Also could be an opportunity for them to have some fun: https://www.creativebloq.com/web-design/best-404-pages-812505

Then, we need to get that language translated.

Reply:
Putting all the languages on a single 404 page could be fun and seems like would make things easier

Reply:
#resistance 

![image](https://user-images.githubusercontent.com/5697474/36829878-b3d559b2-1ce6-11e8-926f-a2220a727753.png)